src/test/java/uk/gov/pay/api/it/telephone/CreateTelephonePaymentIT.java

```diff
@@ -126,6 +126,9 @@ public void createTelephonePaymentWithRequiredFields() {
 
     @Test
     public void returnExistingTelephonePayment() {
+        requestBody.put("card_expiry", "01/08");
+        createTelephonePaymentRequest
+                .withCardExpiry("01/08");
```

Comment:
Why are we doing this? This integration test doesn't use optional fields so no need to pass along card_expiry. The mock can be modified to not return cardExpiry.